helpers/import.sh

```diff
@@ -0,0 +1,170 @@
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[4]" "$ORG_ID roles/cloudsql.viewer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[5]" "$ORG_ID roles/compute.networkViewer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[6]" "$ORG_ID roles/iam.securityReviewer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[7]" "$ORG_ID roles/orgpolicy.policyViewer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[8]" "$ORG_ID roles/servicemanagement.quotaViewer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[9]" "$ORG_ID roles/serviceusage.serviceUsageConsumer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_write[0]" "$ORG_ID roles/compute.securityAdmin serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_compute_firewall.forseti-server-deny-all" "$PROJECT_ID/forseti-server-deny-all-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_compute_firewall.forseti-server-ssh-external" "$PROJECT_ID/forseti-server-allow-ssh-external-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_compute_firewall.forseti-server-allow-grpc" "$PROJECT_ID/forseti-server-allow-grpc-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_storage_bucket.server_config" "$PROJECT_ID/forseti-server-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_storage_bucket.cai_export" "$PROJECT_ID/forseti-cai-export-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_compute_instance.forseti-server" "$PROJECT_ID/us-central1-c/forseti-server-vm-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_sql_database_instance.master" "$PROJECT_ID/forseti-server-db-$RESOURCE_NAME_SUFFIX"
```

Comment:
Will need to change these to the cloudsql sub module.

Reply:
Good catch @hshin-g! I have updated these.